### DISCUSSION TITLE
For the electrode position, how in the code by entering xyz coordinates, there is also no position point T6

### DISCUSSION THREAD

**Discussion by xiberJ** [simnibs/simnibs#149] [8. Simulation workflows]
![1684753845716](https://github.com/simnibs/simnibs/assets/122353886/e1de3c7a-30f6-4567-bdef-2a2e65e4c4bc)
![1684753860607](https://github.com/simnibs/simnibs/assets/122353886/375ab52d-426a-406d-aace-aaced3f53a06)

Thanks for taking time out of your busy schedule

**Comment by xiberJ** [comment]
I've also seen a variety of positions for eeg_positions. The default is EEG10-10_UI_Jurak_2007. How do you change other positions

**Comment by oulap** [comment]
There's a few electrode layouts implemented in Simnibs, see here:

https://github.com/simnibs/simnibs/tree/master/simnibs/resources/ElectrodeCaps_MNI

Those get automatically warped to the subject scan and placed into m2m_<subid>/eeg_positions and you can change between them by changing the name you pass to the simulation script. Not sure any of the caps have T6 by default, but if you now its location in MNI space, you can just add it to the .csv files inside the resources-folder. 

In the ElectrodeCaps_MNI folder the xyz-coordinates are the electrode positions in MNI-space, whereas the ones inside eeg_positions are the subject-specific coordinates. The transformation used to warp the caps is stored in m2m<subid>/toMNI

So basically if you have the T6 coordinate in MNI space you could also use the mni2subject_coords command to map the T6 coordinate to your subject, but it's probably easier to define it in one of the caps so it happens automatically.

**Comment by xiberJ** [comment]
![1684761266747](https://github.com/simnibs/simnibs/assets/122353886/e0e9ffaa-d5bd-4d07-b78f-1fab5207a3a5)
I see that there are a variety of eeg_positions, and the default position points in EEG10-10_UI_Jurak_2007. csv are used when running through code. How do I use the other.csv positions
![1684761353649](https://github.com/simnibs/simnibs/assets/122353886/891929e3-d5be-4f77-aa21-4ae5e1d7408a)
Are the coordinates of electrode positions in mni space in the red box? If I know the coordinates of T6, I just need to format it in this folder, right?

**Comment by oulap** [comment]
If you're using the GUI to do the simulation you can change the cap by selecting "Edit" -> "Select EEG Cap" and then clicking on the cap you want. If you're scripting the simulations you can change the field:

S.eeg_cap = './m2m_ernie/eeg_positions/EEG10-10_Cutini_2011.csv'

to point to whichever cap you prefer. 

The coordinates of the electrodes are in MNI coordinates *only* inside <path_to_simnibs>/simnibs/resources/ElectrodeCaps_MNI

So if you add a row defining the position of T6 in MNI coordinates to one of those csv files that should work.

The eeg cap files inside the m2m-folder are already mapped to the space of the subjects and are not in MNI space.
